install/kubernetes/helm/istio/charts/grafana/dashboards/galley-dashboard.json

```diff
@@ -494,14 +494,14 @@
           "refId": "A"
         },
         {
-          "expr": "istio_mcp_clients_total{component=\"galley\"}",
+          "expr": "galley_mcp_source_clients_total",
```

Comment:
why did this change? did we change the metrics in this PR or was it wrong before?

Reply:
istio/installer and istio/istio charts were out of sync. istio/istio was correct but the patch was reversed. I'll revert this and make the correction to istio/installer charts.

Reply:
See https://github.com/istio/installer/pull/658